### PR TITLE
Move save to toolbar

### PR DESCRIPTION
**Pull request by ClementPasteau**
image
image
image

### CODE REVIEW

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageHeader.js [hunk 1]
```diff
@@ -39,16 +44,32 @@ export const HomePageHeader = ({
...
+                size="small"
+                id="main-toolbar-project-manager-button"
+                onClick={onOpenProjectManager}
+                tooltip={t`Project Manager`}
+                color="default"
+                disabled={!hasProject}
+              >
+                <ProjectManager />
+              </IconButton>
+              {!!hasProject && (
```
Comment (D8H) [inline newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageHeader.js:59]: Is the `!!` needed?

**Review by 4ian — APPROVED** [review]

**Comment by tristanbob** [comment]
This save icon looks different than what people expect. I'd love to see a more standard icon.

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageHeader.js (hunk 1 above)
Comment (ClementPasteau) [reply newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageHeader.js:59]: probably not, but I think it doesn't hurt having it!

**Comment by ClementPasteau** [comment]
@tristanbob the floppy disk? isn't it what's used in most apps?

**Pull request merged**